runtime/src/genesis_config_presets.rs

```diff
@@ -93,78 +94,57 @@ pub fn authority_keys_from_seed(
     )
 }
 
-/*
 // Generate authority IDs from SS58 addresses.
 pub fn authority_ids_from_ss58(
-    sr25519_key: &str,
+    sr25519_account_key: &str,
+    sr25519_common_key: &str,
     ed25519_key: &str,
```

Comment:
Using a name like `session_key` might make the meaning of this key more clear?
```suggestion
    sr25519_session_key: &str,
    ed25519_session_key: &str,
```